catalog/confluence/catalog-confluence-source/pom.xml

```diff
@@ -116,13 +116,14 @@
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>ddf.security</groupId>
-            <artifactId>ddf-security-common</artifactId>
+            <groupId>ddf.lib</groupId>
+            <artifactId>gson-support</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>io.fastjson</groupId>
-            <artifactId>boon</artifactId>
```

Comment:
✏️ Boon can be removed from the dependency list in the documentation. https://github.com/codice/ddf/blob/7f61ab9478cc54fe1ddf6a1c701f98c84965ce0b/distribution/docs/src/main/resources/content/_reference/_dependencyList/ddf-dependency-list.adoc#L71